### PR TITLE
ready to merge: Kristina's portfolio

### PR DESCRIPTION
**Pull request by kristinabrown**

### CODE REVIEW

**Comment by jcasimir** [comment]
You're the best :+1:

File: students/kristina_brown/20150814-kristina_brown-m4.markdown [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* JavaScript Style: 3
+
+#### Scale Up
+
+* [GitHub URL](https://github.com/kristinabrown/scale-up)
+* [Original Assignment](https://github.com/JumpstartLab/curriculum/blob/master/source/projects/the_scale_up.markdown)
+
+In this seven-day project I took a pivoted project, added a million rows in the data base, and improved its performace. It was evaluated by Steve with the following results:
+
+"I mean, it seems like you know what you're doing."
```
Comment (worace) [inline students/kristina_brown/20150814-kristina_brown-m4.markdown]: this is my assessment as well

**Comment by loriculberson** [comment]
Kristina the workhorse!!!! You're work ethic and speed are remarkable. It was a pleasure to work with you. I learned a lot, laughed a ton and am glad I got to meet you. When I grow up, I want to be you!!!

**Comment by patrickmedaugh** [comment]
Kristina, I think all of 1502 looks up to you. You are infuriatingly good at coding already and really so great to work with. The real question is: what can't you do? You learn quickly, work quickly and do everything the right way. It's been such a pleasure getting to know you these past months. Best of luck.

**Comment by boomkenster** [comment]
Pros.
Great having Kristina in the class a rabbit to catch and role model to be.(cleaning, helping students and following the rules). She is incredible in her abilities, judgement and persona. 

Cons
 There was one day she didn't wash dishes. She can't stop biting her nails and she pronounces my name "Kinney" ( like Steve's last name)

**Pull request merged**